ui/fmradio.c

```diff
@@ -38,15 +38,20 @@ void UI_DisplayFM(void)
 	UI_PrintString(String, 0, 127, 0, 12);
 
 	memset(String, 0, sizeof(String));
-
+	strcpy(String, text);
 	UI_PrintString(String, 0, 127, 2, 10);
 
 	memset(String, 0, sizeof(String));
 
 	sprintf(String, "%3d.%d", gEeprom.FM_FrequencyPlaying / 10, gEeprom.FM_FrequencyPlaying % 10);
-	UI_DisplayFrequency(String, 32, 4, true);			
+	UI_DisplayFrequency(String, 32, 4, true);
 
 	ST7565_BlitFullScreen();
 }
 
+void UI_DisplayFM(void)
+{
+	UI_DisplayFM_text("");
```

Comment:
Whereas I like this improvement, the implementation is not slim with the wrapper function approach. And considering we're at 99% mark of size utilization it is quite important.

It could be rewritten to use a global boolean (which probably is already defined, but not used) to indicate scan in progress and then the ui function can determine whether or not display `SCN`.